app/models/manageiq/providers/vmware/infra_manager/inventory/parser/virtual_machine.rb

```diff
@@ -384,7 +384,7 @@ def parse_virtual_machine_labels(vm, _props)
         )
       end.compact
 
-      persister.tag_mapper.map_labels("VmOrTemplate", persister_labels).each do |tag|
+      persister.tag_mapper.map_labels("VmVmware", persister_labels).each do |tag|
```

Comment:
This was added here: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/ops_controller/settings/label_tag_mapping.rb#L33-L34